internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,77 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	input := strings.Split(data, ",")
+	if len(input) != 2 {
+		return 0, 0, fmt.Errorf("неверный формат данных")
+	}
+	stepsStr := input[0]
+	durationStr := input[1]
+
+	// Проверка на лишние пробелы до и после значений
+	if strings.Contains(stepsStr, " ") || strings.Contains(durationStr, " ") {
+		return 0, 0, fmt.Errorf("некорректный формат данных")
+	}
+
+	// Удаляем пробелы внутри значения
+	stepsStr = strings.TrimSpace(stepsStr)
+	durationStr = strings.TrimSpace(durationStr)
+
+	// Парсим количество шагов
+	steps, err := strconv.Atoi(stepsStr)
+	if err != nil {
+		return 0, 0, fmt.Errorf("некорректное количество шагов")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```